site/sfguides/src/automated-query-performance-insights-with-streamlit/automated-query-performance-insights-with-streamlit.md

```diff
@@ -0,0 +1,205 @@
+author: Chanin Nantasenamat
+id: automated-query-performance-insights-with-streamlit
+summary: Build an interactive Streamlit app within Snowflake Notebooks to analyze query performance and identify optimization opportunities. This guide demonstrates how to create visualizations and insights for query execution times.
+categories: streamlit, notebooks, snowflake, featured
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Streamlit, Snowflake, Python, Data Analysis, Query Performance
+
+# Building an Interactive Query Performance Analysis App with Streamlit in Snowflake Notebooks
```

Comment:
@sfc-gh-cnantasenamat I'd recommend shortening the title "Build Interactive Query Performance App in Snowflake"